src/OpenTracingMock/Span.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace OpenTracingMock;
+
+use OpenTracing\Span as OTSpan;
+
+final class Span implements OTSpan
+{
+    /**
+     * @var string
+     */
+    private $operationName;
+
+    /**
+     * @var SpanContext
+     */
+    private $context;
+
+    /**
+     * @var array
+     */
+    private $tags;
```

Comment:
This should be set default to an empty array.

This function will error out if not.
    ```public function setTags(array $tags)
    {
        $this->tags = array_merge($this->tags, $tags);
     }```